src/main/java/org/openstreetmap/atlas/geography/atlas/packed/PackedTagStore.java

```diff
@@ -142,6 +191,24 @@ public String get(final long index, final String key)
         return null;
     }
 
+    @Override
+    public ProtoAdapter getProtoAdapter()
+    {
+        return new ProtoPackedTagStoreAdapter();
+    }
+
+    @Override
+    public int hashCode()
+    {
+        final int initialPrime = 31;
+        final int hashSeed = 37;
+
+        int hash = initialPrime;
```

Comment:
instead of assigning ```initialPrime``` to ```hash``` how about

```
final int initialPrime = 31;
final int hashSeed = 37;

int hash = hashSeed * initialPrime + this.keys.hashCode();
hash = hashSeed * hash + this.values.hashCode();

return hash;
```

Reply:
This makes a lot more sense haha.